core/storage/email/gae_models.py

```diff
@@ -116,6 +141,18 @@ def get_by_hash(cls, email_hash, sent_datetime_lower_bound=None):
         which is a datetime instance. If this is given, only
         SentEmailModel instances sent after sent_datetime_lower_bound
         should be returned.
+
+        Args:
+            email_hash: str. The hash value of the email.
+            sent_datetime_lower_bound: Date and time. The lower bound on 
+            sent_datetime of the email to be searched.
+
+        Returns:
+            The message searched given the hash value of email.
```

Comment:
Add type information. Also, I'm not sure the description you've given is correct; there seem to be multiple messages being returned.